.contrib/Parser/DATAS/02 - Outdoor Zones/02 Eastern Kingdoms/Elwynn Forest.lua

```diff
@@ -1988,7 +1988,7 @@ root(ROOTS.Zones, m(EASTERN_KINGDOMS, {
 							["timeline"] = { "added 4.0.3.13287" },
 						}),
 						i(132094, {	-- Proprietor's Mitts
-							["timeline"] = { "added 7.0.3.22248" },
+							["timeline"] = { "added 4.0.3.13287" },
```

Comment:
This itemID is much too high to have been added during 4.0.3

Reply:
ya my bad, head in ass moment. I reverted it back. Im running a human through just to confirm its not a human only item. It threw me off when the added patch was greater then the quest removed patch.

Reply:
Yeah, you'll notice several removed quests had items retroactively added as rewards for them with the 7.0.3 and 7.3.5 patches when they introduced Plate and Mail items for various lower level quests.